release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -564,10 +564,6 @@ An {product-title} cluster with multi-architecture compute machines supports com
 * xref:../installing/installing_aws/upi/installing-aws-multiarch-support-upi.adoc#installing-a-cluster-with-multiarch-support_upi-aws-multiarch-support[Installing a cluster with multi-architecture support ({aws-short}: User-provisioned infrastructure)]
 * xref:../installing/installing_gcp/installing-gcp-multiarch-support.html#installing-a-cluster-with-multiarch-support_installing-gcp-multiarch-support[Installing a cluster with multi-architecture support ({gcp-short})]
 
-==== Deploy {azure-short} in the Spain region
```

Comment:
@dfitzmau, did you mean to remove this RN?

Reply:
Thanks for the comment. Yes, as it's a dupe. See the following note: 
![Screenshot from 2024-09-30 13-54-25](https://github.com/user-attachments/assets/ebc1d3e0-b76f-4ff6-9b86-cf4a83c4de0a)